guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-satellite-server.adoc

```diff
@@ -12,74 +11,20 @@ As a result, `{foreman-installer}` fails to execute while enabling features or u
 <2> Path to the private key that was used to sign {ProjectServer} certificate.
 <3> Path to the Certificate Authority bundle.
 +
-If the command is successful, it returns two `{foreman-installer}` commands, one of which you must use to deploy a certificate to {ProjectServer}.
-ifdef::satellite[]
-+
-.Example output of `katello-certs-check`
-[options="nowrap", subs="+quotes,attributes"]
-----
-Validation succeeded.
-
-To install the Red Hat Satellite Server with the custom certificates, run:
-
-  {foreman-installer} --scenario satellite \
-    --certs-server-cert "_/root/{project-context}_cert/{project-context}_cert.pem_" \
-    --certs-server-key "_/root/{project-context}_cert/{project-context}_cert_key.pem_" \
-    --certs-server-ca-cert "_/root/{project-context}_cert/ca_cert_bundle.pem_"
-
-To update the certificates on a currently running Red Hat Satellite installation, run:
-
-  {foreman-installer} --scenario satellite \
-    --certs-server-cert "_/root/{project-context}_cert/{project-context}_cert.pem_" \
-    --certs-server-key "_/root/{project-context}_cert/{project-context}_cert_key.pem_" \
-    --certs-server-ca-cert "_/root/{project-context}_cert/ca_cert_bundle.pem_" \
-    --certs-update-server --certs-update-server-ca
-----
-endif::[]
-ifndef::satellite[]
-+
-.Example output of `katello-certs-check`
-[options="nowrap", subs="+quotes,attributes"]
-----
-Validation succeeded.
-
-To install the Katello main server with the custom certificates, run:
-
-  foreman-installer --scenario katello \
-    --certs-server-cert "_/root/{project-context}_cert/{project-context}_cert.pem_" \
-    --certs-server-key "_/root/{project-context}_cert/{project-context}_cert_key.pem_" \
-    --certs-server-ca-cert "_/root/{project-context}_cert/ca_cert_bundle.pem_"
-
-To update the certificates on a currently running Katello installation, run:
-
-  foreman-installer --scenario katello \
-    --certs-server-cert "_/root/{project-context}_cert/{project-context}_cert.pem_" \
-    --certs-server-key "_/root/{project-context}_cert/{project-context}_cert_key.pem_" \
-    --certs-server-ca-cert "_/root/{project-context}_cert/ca_cert_bundle.pem_" \
-    --certs-update-server --certs-update-server-ca
-----
-endif::[]
-+
-Note that you must not access or modify `/root/ssl-build`.
-. From the output of the `katello-certs-check` command, depending on your requirements, enter the `{foreman-installer}` command that installs a new {Project} with custom SSL certificates or updates certificates on a currently running {Project}.
-+
-If you are unsure which command to run, you can verify that {Project} is installed by checking if the file `/etc/foreman-installer/scenarios.d/.installed` exists.
-If the file exists, run the second `{foreman-installer}` command that updates certificates.
-+
 [IMPORTANT]
 ====
 `{foreman-installer}` needs the certificate archive file after you deploy the certificate.
```

Comment:
This note repeats the note that's above the procedure.

Also, the instructions don't talk about an archive file but rather about a set of individual files.

I think it's better to tell the user ahead of time and the exact text of that note is IMHO better so would you agree this should be dropped?